### ISSUE TITLE
Themes section

### ISSUE THREAD

**Issue by pH142857** [gohugoio/hugoDocs#2355]
I was surprised that gohugo.io didn't include any tutorial about themes, so I did a google search and I found what seems to be a brazilian mirror of the docs where there is every docs section, plus a Themes one: https://gohugobrasil.netlify.app/themes/customizing/
How come it's not on the main site and shouldn't it be?

**Comment by jmooring** [comment]
Yeah, the entire section was removed several years ago with this commit:
<https://github.com/gohugoio/hugoDocs/commit/3fdb45b3cda8e42f5822bb1d44578a28c8de4b8f> 

Not sure why.

Restoring this, or something like it, has been on my to-do list for a while.  Thanks for creating the issue.

> brazilian mirror of the docs where there is every docs section

Regarding these mirrors:
- They were created by a third-party
- They are not maintained by the Hugo project team
- They are, from what I have seen, quite old (which in this case is a good thing!)
